javascript-modal/styles.css

```diff
@@ -0,0 +1,71 @@
+.container {
+  background-color: white;
+  position: fixed;
+  left: 0;
+  right: 0;
+  top: 0;
+  bottom: 0;
+}
```

Comment:
Your `.container` shouldn't be `position: fixed`. Only the modal should

Reply:
@shawnkost When I remove the position prop and the offset props, it doesn't change anything visibly. 
What would this affect?

Reply:
@sarah-you It should change something visibly! At least it does on my end! You don't want the `.container` to be position fixed, because then if the page was long enough, it would scroll down with you. We just want this button to stay in the normal document flow, and stay at the top.